timezone_converter/comparison_view.py

```diff
@@ -46,7 +46,7 @@ def _get_headers(self) -> List[str]:
 
         return headers
 
-    def _build_table(self) -> Table:
+    def _build_table(self, single_hour) -> Table:
```

Comment:
As suggested on main.py, we would use it as instance argument, so no need to pass it here.